contracts/Gate.sol

```diff
@@ -151,19 +164,24 @@ contract Gate is IGate, Ownable, Pausable {
      *
      * @param _tokenIdSupply an ID of a supply token (ERC-1155) [voucherSetID]
      * @param _conditionalTokenId an ID of a conditional token
+     * @param _condition condition that will be checked when a user commits using a conditional token
      */
-    function registerVoucherSetId(uint256 _tokenIdSupply, uint256 _conditionalTokenId)
+    function registerVoucherSetId(uint256 _tokenIdSupply, uint256 _conditionalTokenId, Condition _condition)
         external
         override
         whenNotPaused
         onlyRouterOrOwner
     {
         require(_conditionalTokenId != 0, "TOKEN_ID_0_NOT_ALLOWED");
```

Comment:
I think this should be removed now, since tokenID 0 has a different meaning now.

Reply:
Removed. Will be included in next commit.